Sources/ObjCSupport/Builders/LOKBaseLayoutBuilder.m

```diff
@@ -1,29 +0,0 @@
-// Copyright 2018 LinkedIn Corp.
-// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-
-#import "LOKBaseLayoutBuilder.h"
-#import "LOKInsetLayoutBuilder.h"
-
-@implementation LOKBaseLayoutBuilder
-
-- (id<LOKLayout>)build {
-    NSAssert(NO, @"This method is expected to be overridden by a superclass.");
-    return nil;
-}
-
-- (id<LOKLayout>)layout {
```

Comment:
Do you think if we should add a protocol to have `- (id<LOKLayout>)layout` and `BaseLayout` elements, like `alignment`, `flexibility`, `viewReuseId`, `viewClass`. So any layout builder should implement those for preventing different variants of APIs.

Reply:
Yeah. That would be good. But how can we do that, because the return value of all of these things are a specific type.

Reply:
Resolved in person. Even though the protocol will declare `id<LOKLayout>` and `id<LOKLayoutBuilder>` as the types, when we implement the protocol, we can use the concrete type and it will satisfy the requirement. I will make these changes now.

Reply:
Thanks 👍